src/frontend/src/components/SearchInput/SearchInput.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  input: {
+    fontSize: '1.6rem',
+    '&:hover': {
+      border: '2px solid',
+      borderColor: theme.palette.background.default,
+    },
+    '&:focus': {
+      border: '2px solid',
+      borderColor: theme.palette.primary.main,
+    },
+    '& > *': {
+      fontSize: '1.6rem !important',
+      color: theme.palette.text.default,
+    },
+    height: '55px',
+    backgroundColor: theme.palette.background.default,
+    paddingLeft: '10px',
+    // The border around the textField inputs was a default of the type so I had to add my own
+    border: '1px solid #B3B6B7',
+    borderRadius: '7px',
+  },
+}));
+
+function AuthorSearchInput(props) {
+  const { text, onChange, filter } = props;
+  const classes = useStyles();
+  return (
+    <>
+      <input
+        list="searchData"
+        className={classes.input}
+        placeholder="How to Get Started in Open Source"
+        inputProps={{ 'aria-label': 'search telescope' }}
+        value={text}
+        onChange={onChange}
+        filter={filter}
+      />
+      <datalist id="searchData">
+        <option>Test 1</option>
+        <option>Test 2</option>
+        <option>Test 3</option>
+      </datalist>
+    </>
+  );
+}
+
+function PostSearchInput(props) {
+  const { text, onChange, filter } = props;
+  const classes = useStyles();
+  return (
+    <input
```

Comment:
Previously this was:

```
 <TextField
  className={classes.input}
  placeholder="How to Get Started in Open Source"
  inputProps={{ 'aria-label': 'search telescope' }}
  variant="outlined"
  value={text}
  onChange={(event) => onTextChange(event.target.value)}
/>
```

Should we just use that again?  Why switch to `<input>`?  If it's going to over complicate things to switch, leaving with `<input>` for now is also fine.

Reply:
Makes sense. I've switch it back to a textField.

Reply:
Actually PostSearchInput.jsx throws an error when using TextField when I switched it over to its own file. I've imported the TextField component. Strange. For now I'll leave it as an input.